README.md

```diff
@@ -59,26 +59,26 @@ all discussion should happen in the [php-cds/php-cds-discussion](https://github.
 
 Discussion in a separate repository prevents noise for those who are only interested in voting.
 
-Those who want to be more involved in the community driven standards process can watch the [php-cds/php-cds-discussion](https://github.com/php-cds/php-cds-discussion) repository.
+Those who want to be more involved in the Community-driven Standards process can watch the [php-cds/php-cds-discussion](https://github.com/php-cds/php-cds-discussion) repository.
```

Comment:
'Community-Driven Standards' is a proper term representative of an entity. 'community-driven standards' is a descriptive phrase that does not carry the connotation of an entity. It's your choice, but I see no particular reason for proper capitalization.